tests/nlu/base/test_utils.py

```diff
@@ -61,42 +76,30 @@ def test_ordered():
     assert ordered(target) == [("a", [1, 2, 3]), ("b", 1), ("c", "a")]
 
 
-@pytest.mark.parametrize(
-    ("model_dir", "expected"),
-    [
-        ("test_models/test_model_mitie/model_20170628-002704", True),
-        ("test_models/test_model_mitie_sklearn/model_20170628-002712", True),
-        ("test_models/test_model_spacy_sklearn/model_20170628-002705", True),
-        ("test_models/", False),
-        ("test_models/nonexistent_for_sure_123", False),
-    ],
-)
-def test_is_model_dir(model_dir, expected):
-    assert is_model_dir(model_dir) == expected
+def test_empty_is_model_dir(empty_model_dir):
+    assert is_model_dir(empty_model_dir)
 
 
-def test_is_model_dir_empty(empty_model_dir):
-    assert is_model_dir(empty_model_dir)
+def test_non_existent_folder_is_no_model_dir():
+    assert not is_model_dir("nonexistent_for_sure_123/")
 
 
-def test_remove_model_empty(empty_model_dir):
-    assert remove_model(empty_model_dir)
+def test_data_folder_is_no_model_dir():
+    assert not is_model_dir("data/")
 
 
-def test_remove_model_with_files(empty_model_dir):
-    metadata_file = "metadata.json"
-    metadata_content = {"pipeline": "pretrained_embeddings_spacy", "language": "en"}
-    metadata_path = os.path.join(empty_model_dir, metadata_file)
-    write_json_to_file(metadata_path, metadata_content)
+def test_model_folder_is_model_dir(fake_model_dir):
+    assert is_model_dir(fake_model_dir)
 
-    fake_obj = {"Fake", "model"}
-    fake_obj_path = os.path.join(empty_model_dir, "component.pkl")
-    with io.open(fake_obj_path, "wb") as f:
-        pickle.dump(fake_obj, f)
 
+def test_remove_model_empty(empty_model_dir):
     assert remove_model(empty_model_dir)
 
 
+def test_remove_model_with_files(fake_model_dir):
+    assert remove_model(fake_model_dir)
```

Comment:
are these 2 (empty and fake) actually the same path?

Reply:
what do you mean same path?

Reply:
just am confused because the fake model dir is created from the empty model dir (`not empty anymore`) but then you remove the empty model dir before attempting to remove the fake model dir -- probably that these 2 are completely separate and i'm just thinking in the integration test mindset where everything affects the others